packages/Popover/stories/examples/WithTriggers.js

```diff
@@ -46,14 +45,16 @@ const ExampleStory = () => (
       </h5>
       <Popover isEager>
         <Popover.Trigger>
-          {handler => (
+          {(handler, ariaId) => (
```

Comment:
maybe wrap all attributes in an object and then spread it across the component `{...attributes}` this is how I have seen other API doing it. That way if we want to add more attributes will be simple and transparent for our users